nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/RunNiFi.java

```diff
@@ -581,30 +585,31 @@ private Status getStatus(final Logger logger) {
         return new Status(port, pid, pingSuccess, alive);
     }
 
-    public void status() throws IOException {
+    public int status() throws IOException {
         final Logger logger = cmdLogger;
         final Status status = getStatus(logger);
         if (status.isRespondingToPing()) {
             logger.info("Apache NiFi is currently running, listening to Bootstrap on port {}, PID={}",
                     new Object[]{status.getPort(), status.getPid() == null ? "unknown" : status.getPid()});
-            return;
+            return 0;
         }
 
         if (status.isProcessRunning()) {
             logger.info("Apache NiFi is running at PID {} but is not responding to ping requests", status.getPid());
-            return;
+            return 0;
```

Comment:
@michalklempa community seems to prefer returning 4. 

http://mail-archives.apache.org/mod_mbox/nifi-dev/201610.mbox/%3CCAJpGy9Jp8PyhhAXueH8x%2BQ4y-JX-VvxbU82M-bLbuhDHSzP_Fg%40mail.gmail.com%3E

Mind adjusting? I will merge it after.